hca/upload/cli/list_areas_command.py

```diff
@@ -4,12 +4,15 @@
 
 
 class ListAreasCommand:
```

Comment:
This should be fixed now. In the new CLI entry point dispatch convention, the return value of the call to `entry_point` is printed on stdout if it's a string or bytes, or printed as JSON if it's a mapping. The upload commands are using a class constructor call (__new__/__init__), which returns an instance of the command class, which is not something that should be printed. I special-cased this in the entry point dispatcher to ignore the return value in this case. Alternatively, we could change the Command classes to move the command handler logic from `__init__` to `__call__`, and change the entry points to be instances of the Command class instead of the classes themselves.